README.md

```diff
@@ -64,7 +64,7 @@ No modules.
 | [aws_security_group_rule.orchestrator_agent_egress_rule](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group_rule) | resource |
 | [aws_security_group_rule.orchestrator_agent_ingress_rule](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group_rule) | resource |
 | [aws_region.current_region](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/region) | data source |
-| [template_file.orchestrator_agent_container_definitions](https://registry.terraform.io/providers/hashicorp/template/latest/docs/data-sources/file) | data source |
+source |
```

Comment:
This escaped the chopping block 😄 

Mind getting rid of line 44 also, that's the other mention of the template provider

Reply:
> This escaped the chopping block 😄
> 
> Mind getting rid of line 44 also, that's the other mention of the template provider

Oops! Done.